packages/errors/test/unit/visualSnapshotErrors_spec.ts

```diff
@@ -719,6 +730,13 @@ describe('visual error templates', () => {
         withSystemError: [aggregateErrorWithSystemError],
       }
     },
+    CLOUD_PROTOCOL_UPLOAD_UNKNOWN_ERROR: () => {
+      const error = new Error('some message')
```

Comment:
Will this be updated to be a more specific error message?

Reply:
This is for the visual snapshotting of errors, so all it's testing is that the error message matches the snapshot based on the interpolation - I've modified it to use `makeErr` like some of the other snapshot setups.

This is for upload errors that have not yet been categorized, so we don't quite have a more specific error to supply for the snapshots

Reply:
changed to use a more standard default error message here in https://github.com/cypress-io/cypress/pull/30235/commits/d2cfbb3862aa589338b759031ccd98b451febcb8